ps_qualityassurance.php

```diff
@@ -68,7 +76,20 @@ public function install()
             'UNIQUE KEY `name` (`name`)' .
             ') ENGINE=' . _MYSQL_ENGINE_ . ' DEFAULT CHARSET=UTF8;';
 
-        return Db::getInstance()->execute($query) && parent::install();
+        $query2 = 'CREATE TABLE IF NOT EXISTS `' . _DB_PREFIX_ . 'quality_assurance_hook_logs` (' .
+            '`id` int(11) UNSIGNED NOT NULL AUTO_INCREMENT,' .
+            '`request_identifier` varchar(255) NOT NULL,' .
+            '`hook_name` varchar(255) NOT NULL,' .
+            '`hook_parameters` text NOT NULL,' .
+            '`output` text NOT NULL,' .
+            '`called_at` datetime NOT NULL,' .
+            '`error` TINYINT(1) NOT NULL,' .
+            'PRIMARY KEY (`id`)' .
+            ') ENGINE=' . _MYSQL_ENGINE_ . ' DEFAULT CHARSET=UTF8;';
+
+        return Db::getInstance()->execute($query1)
```

Comment:
```suggestion
        return Db::getInstance()->execute($query)
```

Reply:
Fixed in https://github.com/PrestaShop/ps_qualityassurance/pull/8/commits/da03ba8a722b9e6ec2da896b72f65667ddc7c0a4